.env.development

```diff
@@ -8,3 +8,5 @@ VUE_APP_GOOGLE_CLIENT_ID = 885568627115-2lu9inbk9n7o1hc0me6dip2ac5drprj0.apps.go
 VUE_APP_GIT_CLIENT_ID = 346108128ad51a5eccd1
 
 VUE_APP_KDF_PARAMS_N = 1024
+
+VUE_APP_EXTRA_TIMEOUT_FOR_CLIENT_IDS=['bf86799a-4d6d-4d7c-892a-e5559814aae1','1a23cc96-fcd8-4ed0-bdb3-038dd41e57e8']
```

Comment:
Does `dotenv` correctly work with arrays?

Reply:
yes, it works